readme.md

```diff
@@ -34,16 +34,16 @@ gulp.task('default', function () {
 
 ##### ui
 
-Type: `string`  
```

Comment:
Thanks, thought I was but apparently not - easy to fix.  Doesn't seem like it much matters though, as this was outright closed with no justification.